src/tests/electron/tests/android-setup-prompt-choose-device.test.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    leftFooterButtonAutomationId,
+    rightFooterButtonAutomationId,
+} from 'electron/views/device-connect-view/components/android-setup/android-setup-step-layout';
+import { deviceDescriptionAutomationId } from 'electron/views/device-connect-view/components/android-setup/device-description';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { createApplication } from 'tests/electron/common/create-application';
+import { scanForAccessibilityIssuesInAllModes } from 'tests/electron/common/scan-for-accessibility-issues';
+import { AndroidSetupViewController } from 'tests/electron/common/view-controllers/android-setup-view-controller';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import {
+    commonAdbConfigs,
+    delayAllCommands,
+    MockAdbConfig,
+    setupMockAdb,
+    simulateServiceNotInstalled,
+} from '../../miscellaneous/mock-adb/setup-mock-adb';
+
+const [closeId, nextId, rescanId] = [
+    leftFooterButtonAutomationId,
+    rightFooterButtonAutomationId,
+    'rescan', // TODO: Why doesn't rescanAutomationId work here?
```

Comment:
the one existing rescanAutomationId I see is in prompt-connected-start-testing-step.tsx, and its value is specific to that step. I wonder if we should consolidate the automation ids and export them into the various tsx steps & the various e2e tests. Another option is adding a rescanAutomationId to this specific dialog and using that explicitly

Reply:
Fixed by exporting rescanAutomationId from prompt-choose-device and using it here